services/static-webserver/client/source/class/osparc/auth/LoginPage.js

```diff
@@ -114,11 +114,15 @@ qx.Class.define("osparc.auth.LoginPage", {
 
       pages.add(login);
       const config = osparc.store.Store.getInstance().get("config");
-      if (config["invitation_required"]) {
-        if (osparc.product.Utils.getProductName().includes("s4l")) {
-          // all S4Ls
-          pages.add(requestAccount);
-        }
+      if (config["invitation_required"] &&
+        (
+          osparc.product.Utils.isProduct("s4l") ||
+          osparc.product.Utils.isProduct("s4lacad") ||
+          osparc.product.Utils.isProduct("s4ldesktop") ||
+          osparc.product.Utils.isProduct("s4ldektopacad")
+        )
+      ) {
```

Comment:
At least there is one more instance of this combination

Reply:
And that's the second and last :)